asana/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,22 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.0.0
+name: Asana
+displayName: Asana
+createdAt: 2022-05-16T00:00:00Z
+description: Asana integration for Keptn
+logoURL: https://upload.wikimedia.org/wikipedia/commons/3/3b/Asana_logo.svg
+digest: 2022-05-16T00:00:00Z
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+  - keptn
+  - asana
+links:
+  - name: Source
+    url: https://asana.com
+recommendations:
+  - url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: "webhook"
+  keptn/version: "0.10.0-0.15.0"
```

Comment:
Are these Keptn versions still up to date or is it also compatible with newer versions?